internal/services/network/virtual_hub_connection_resource_test.go

```diff
@@ -279,6 +279,29 @@ func TestAccVirtualHubConnection_routeMapAndStaticVnetLocalRouteOverrideCriteria
 	})
 }
 
+func TestAccVirtualHubConnection_routeMapAndStaticVnetPropagateStaticRoutes(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_virtual_hub_connection", "test")
+	r := VirtualHubConnectionResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			// Apply the non-default configuration
+			Config: r.routeMapAndStaticVnetPropagateStaticRoutes(data, false),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		{
```

Comment:
Missing the ImportStep here
```suggestion
		},
		data.ImportStep(),
		{
```

Reply:
I thought it is required. But I didn't see that step in the other tests like:
https://github.com/aeimer/terraform-provider-azurerm/blob/feat/28556-propagate-static-routes/internal/services/network/virtual_hub_connection_resource_test.go#L209

Will change it, rerun tests and push it.